CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+* Fix crash in case insensitive query on indexed string columns when nothing matches ([#6836](https://github.com/realm/realm-cocoa/issues/6836), since v6.0.0)
```

Comment:
doesn't this fix apply to case sensitive string equals queries as well?

Reply:
No, I don't think so. The logic in `get_key()` is different.